website/content/docs/nia/api/health.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Consul-Terraform-Sync Health API
+description: >-
+  Consul-Terraform-Sync Health API
+---
+
+# Health
+
+The `/health` endpoints provide information on the health of a running Consul-Terraform-Sync (CTS) instance.
+
+## Get Health
+
+This endpoint returns the health of CTS.
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/health` | `application/json` |
+
+#### Response Statuses
+
+| Status | Reason                                               |
+| ------ | ---------------------------------------------------- |
+| 200    | CTS is healthy                                       |
+| 503    | CTS is unhealthy                                     |
+
+#### Response Fields
+
+| Name         | Type   | Description                                                                        |
+| ------------ | ------ | ---------------------------------------------------------------------------------- |
+| `request_id` | string | The ID of the request. Used for auditing and debugging purposes.                   |
+
+#### Example: Getting the health of CTS
+
+Request:
+
+```shell-session
+$ curl --request GET \
+  localhost:8558/v1/health
+```
+
+Response:
```

Comment:
```suggestion
The following response shows the request ID, indicating that CTS is healthy:

```
I don't know if this statement is true. it's not clear what a healthy response might look like, especially because the example response does not include the `200` code we said was indicative of a healthy CTS.